Document Organizations (#1844)
platformsh/platformsh-docs: docs/src/administration/organizations.md

---
title: "Organizations"
weight: -1
sidebarTitle: "Organizations"
description: |

>   Organizations allow to manage your Platform.sh projects, users and billing.

Reviewer: ```suggestion
  Organizations allow you to manage your Platform.sh projects, users and billing.
```
Chad's suggestion, updated

* **Manage billing** (`billing`): Add, remove and edit billing information. Access invoices and vouchers.
* **Create projects** (`projects:create`): Create new projects within the organization.
* **Manage users** (`users`): Add, remove and edit organization-level users and permissions, including your own.

{{< note theme="warning" title="Important" >}}
A user with the **manage users** permission can add, edit, or remove any user's permissions to manage plans, manage billing or create projects using the Platform.sh CLI.
{{< /note >}}

## Manage organizations with the CLI

You can use the Platform.sh command line interface to manage your organizations.

Available commands (get the full list with 'platform list organization')

```
Available commands for the "organization" namespace:
  organization:billing:address                                  View or change an organization's billing address
  organization:billing:profile                                  View or change an organization's billing profile

>   organization:create                                           Create a new organization

Reviewer: ```suggestion
```

---
title: "Organizations"
weight: -1
sidebarTitle: "Organizations"
description: |

>   Organizations allow to manage Platform.sh projects, users and billing.

Reviewer 2: ```suggestion
  Organizations allow you to manage Platform.sh projects, users and billing.
```

You can use the Platform.sh command line interface to manage your organizations.

Available commands:

* `platform organization:create`
  * Create a new organization.
* `platform organization:info`
  * View or edit an organization.
* `platform organization:list`
  * List all the organizations that you have access to.
* `platform organization:user:list`
  * List all users within an organization.
* `platform organization:user:add`
  * Invite a user to an organization.
* `platform organization:user:get`
  * Get information about a user in an organization.
* `platform organization:user:update`
  * Update permissions of a user in an organization.
* `platform organization:user:delete`
  * Remove a user from an organization.

For example, the following command would invite `alice@example.com` with the **Billing**, **Plans** and **Projects create** permissions to the `my-organization-name` organization.

```bash
platform organization:user:add alice@example.com --org=my-organization-name --permission=billing,plans,projects:create
```

After inviting `alice@example.com`, Alice will receive an invitation email asking to confirm her details and optionally, register for a Platform.sh account.

> To update Alice permissions in your organization, simply run:

Reviewer: ```suggestion
To update Alice's permissions in your organization, run:
```
